### DISCUSSION TITLE
Obsolete CodeQL configuration for javascript?

### DISCUSSION THREAD

**Discussion by Jocke4f** [josephdadams/TallyArbiter#624] [General]
In Security -> Code scanning -> Tool status -> Setup types -> CodeQL code scanning there are two configurations defined as `language:javascript`.

![image](https://github.com/josephdadams/TallyArbiter/assets/39942694/948723f9-efd5-4025-85dc-8db297651d3c)

Is the 2nd one, i.e., 3rd configuration in the list `langugage:javascript` something that should work or something old that should be deleted?

**Comment by Jocke4f** [comment]
The 3rd configuration points to `.github/workflows/codeql-analysis.yml:analyze/language:javascript` which doesn't exist in the workspace. Can this be deleted?

**Comment by josephdadams** [comment]
I deleted it
